tests/test_kernels/test_stationary.py

```diff
@@ -82,7 +82,7 @@ def test_initialization(self, fields: dict, dim: int) -> None:
         assert is_dataclass(self.kernel)
 
         # Input fields as JAX arrays
-        fields = {k: jnp.array([v]) for k, v in fields.items()}
+        fields = {k: jnp.array(v) for k, v in fields.items()}
```

Comment:
backwards-incompatible change due to using proper scalar fields for e.g. variance